### PR TITLE
Update user detail screen connected identities tab

### LINKED ISSUE #331: Update user detail connected identities screen + add identity screens
- [x] add verify / unverify button (force verify, directly change state, not performing verification)
- [x] handle item verified / unverified state
- [x] update item description to include verified
- [x] verify only exist in phone / email

- [x] add identity context menu
- [x] email, phone, username link to corresponding screen
- [x] setup screen for add email, phone, username

### PR DESCRIPTION
**Pull request by axolotl0212**
Closes #331

- update list cell design
- update list structure, one type per list
- add add identity button / menu
- primary identities form
- add email, phone and username screens

### CODE REVIEW

**Comment by axolotl0212** [comment]
# updated connected identities tab 
Screenshot 2020-09-21 at 4 07 58 PM

# primary identities
Screenshot 2020-09-21 at 5 37 10 PM

File: portal/src/graphql/adminapi/AddPhoneScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import { PortalAPIAppConfig } from "../../types";
+
+import styles from "./AddPhoneScreen.module.scss";
+
+interface AddPhoneFormProps {
+  appConfig: PortalAPIAppConfig | null;
+}
+
+// fallback if config does not exist
+const DEFAULT_COUNTRY_CODE = "852";
```
Comment (kiootic) [inline portal/src/graphql/adminapi/AddPhoneScreen.tsx:23]: The default country calling code should not be 852, that kinda arbitrary and biased for international users. Let's follow the default value of config and use first country calling code in lexicographical order: `+1`.

File: portal/src/graphql/adminapi/PrimaryIdentitiesSelectionForm.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import styles from "./PrimaryIdentitiesSelectionForm.module.scss";
+import deepEqual from "deep-equal";
+import NavigationBlockerDialog from "../../NavigationBlockerDialog";
+
+interface PrimaryIdentitiesSelectionFormProps {
+  className?: string;
+  identityLists: IdentityLists;
+}
+
+const PrimaryIdentitiesSelectionForm: React.FC<PrimaryIdentitiesSelectionFormProps> = function PrimaryIdentitiesSelectionForm(
```
Comment (kiootic) [inline portal/src/graphql/adminapi/PrimaryIdentitiesSelectionForm.tsx:17]: FYI we may need to revamp this section after the custom attribute spec is confirmed, as we may have a separate tab to manage it.

cc @louischan-oursky

File: portal/src/graphql/adminapi/UserDetailCommandBar.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        key: "loginAsUser",
+        text: renderToString("UserDetails.command-bar.login-as-user"),
+        iconProps: { iconName: "FollowUser" },
+      },
+
+      {
+        key: "invalidateSessions",
+        text: renderToString("UserDetails.command-bar.invalidate-sessions"),
+        iconProps: {
+          iconName: "CircleAddition",
```
Comment (kiootic) [inline portal/src/graphql/adminapi/UserDetailCommandBar.tsx:37]: It seems this button is gone in the design.

File: portal/src/locale-data/en.json [hunk 4]
```diff
@@ -13,6 +14,10 @@
   "disable": "Disable",
   "confirm": "Confirm",
   "cancel": "Cancel",
+  "disconnect": "Disconnect",
+  "verified": "Verified",
```
Comment (kiootic) [inline portal/src/locale-data/en.json:18]: We have `verified` and `UserDetails.connected-identities.verified`, is there any difference in their semantics?

File: portal/src/graphql/adminapi/UserDetailCommandBar.tsx (hunk 3 above)
Comment (axolotl0212) [reply portal/src/graphql/adminapi/UserDetailCommandBar.tsx:37]: It should be moved to session screen, let's remove it for now and add it to session screen when we update that screen

File: portal/src/locale-data/en.json (hunk 4 above)
Comment (axolotl0212) [reply portal/src/locale-data/en.json:18]: it is duplicated, let's remove `UserDetails.connected-identities.verified` and rename `UserDetails.connected-identities.unverified` to `unverified`

**Comment by axolotl0212** [comment]
rebased and updated @kiootic

**Comment by axolotl0212** [comment]
rebased

**Pull request merged**